docs/docs-site/content/developer/api/rest/_index.md

```diff
@@ -490,7 +490,87 @@ Create a directory at a specific path:
     curl -X POST https://demo.gethue.com/api/v1/storage/mkdir
     
 - **name:** name of the directory
-- **path:** specific path where user wants to create the directory
+- **path:** specific path where you wants to create the directory
+
+### Create File
+
+Create a file at a specific path:
+
+    curl -X POST https://demo.gethue.com/api/v1/storage/touch
+    
+- **name:** name of the file
+- **path:** specific path where you want to create the file
+
+### Rename
+
+Rename a file or directory:
+
+    curl -X POST https://demo.gethue.com/api/v1/storage/rename
+    
+- **src_path:** path of the selected file or directory
+- **dest_path:** path after renaming the selected file or directory
+
+### Move
+
+Move a file or directory to a destination path:
+
+    curl -X POST https://demo.gethue.com/api/v1/storage/move
+    
+- **src_path:** path of the selected file or directory
+- **dest_path:** path at which you want to move the selected file or directory
+
+### Copy
+
+Copy a file or directory to a destination path:
+
+    curl -X POST https://demo.gethue.com/api/v1/storage/copy
+    
+- **src_path:** path of the selected file or directory
+- **dest_path:** path at which you want to save the copy of the selected file or directory
+
+**Note:** On the **Apache Ozone** filesystem, the copy operation returns a string of skipped files if their size is greater than the configured chunk size.
+
+### Get Content Summary
+
+Fetch content summary for a specific file in **HDFS** or **Apache Ozone**:
+
+    curl -X GET https://demo.gethue.com/api/v1/storage/content_summary=/user/hue/weblogs.csv
+
+    curl -X GET https://demo.gethue.com/api/v1/storage/content_summary=ofs://ozone1/testvolume/testbucket/testfile.csv
+
+### Delete
+
+Delete a file or directory:
+
+    curl -X POST https://demo.gethue.com/api/v1/storage/rmtree
+    
+- **path:** path of the selected file or directory
+- **skip_trash:** boolean value to determine moving deleted file or directory to trash or not
```

Comment:
boolean value to indicate whether to move the deleted file or directory to the trash directory. Specify 1 to move the file to the trash.

Please check the technical accuracy of the second sentence.